complaints/forms.py

```diff
@@ -15,13 +15,14 @@ class Meta:
             "avail_start_time",
             "avail_end_time",
         )
-        labels = {
-            "remark": ("Complaint Details"),
-            "contact_no": ("Contact Number"),
-            "room_no": ("Room Number"),
-            "avail_start_time": ("Complaint Details"),
-        }
-        widgets = {"contact_no": forms.Textarea(attrs={"cols": 20, "rows": 1})}
+        
+
+class Handle_form(forms.ModelForm):
```

Comment:
Class names should be `PascalCase`

Reply:
Don't we use Snake case in python and Pascal cases in Java?

Reply:
Not for class names. See [PEP8](http://www.python.org/dev/peps/pep-0008/)